WordPress/Classes/ViewRelated/Blog/BlogDetailHeaderView.m

```diff
@@ -209,4 +214,41 @@ - (void)setUpdatingIcon:(BOOL)updatingIcon
     }
 }
 
+#pragma mark - Drop Interaction Handler
+
+- (BOOL)dropInteraction:(UIDropInteraction *)interaction
+       canHandleSession:(id<UIDropSession>)session API_AVAILABLE(ios(11.0))
+{
+    BOOL isAnImage = [session canLoadObjectsOfClass:[UIImage self]];
+    BOOL isSingleImage = [session.items count] == 1;
+    return (isAnImage && isSingleImage);
+}
+
+- (UIDropProposal *)dropInteraction:(UIDropInteraction *)interaction
+                   sessionDidUpdate:(id<UIDropSession>)session API_AVAILABLE(ios(11.0))
+{
+    CGPoint dropLocation = [session locationInView:self];
+    UIDropOperation dropOperation;
```

Comment:
Nitpicky but, if we initialize with `UIDropOperationCancel` we could avoid the `else` block below.